pkg/tagger/collectors/workloadmeta_test.go

```diff
@@ -1223,112 +1251,6 @@ func Test_mergeMaps(t *testing.T) {
 	}
 }
 
-func TestFargateStaticTags(t *testing.T) {
```

Comment:
These have been copied to `github.com/DataDog/datadog-agent/pkg/util` when `GetStaticTags` was added there by #10952, so they're not necessary here anymore.